VALIDATORS_FAQ.md

```diff
@@ -0,0 +1,245 @@
+
+### What is a delegator?
+
+Delegators are ATOMs holders who  cannot ordo not want to run validator operations, which involve IT admin and secure data center, themselves. Through the Cosmos UI, a user can delegate to a validator and obtain a part of its reward in exchange.
+
+### How to become a validator?
+
+Any participant in the network can signal that they want to become a validator by sending a “self-designation” transaction, where they must fill out the following parameters:
+
+* Validator's name
+* Validator's website
+* Validator's description
+* Initial commission rate
+* Minimum self-bond amount
+* Initial self-bond amount
+
+After that, users can bond ATOMs to this address, effectively adding stake to it. The total stake of an address is the combination of ATOMs bonded by users (called delegators) and ATOMs self-bonded by the participant who designated himself.
+
+Out of all the addresses that signaled themselves, the 100 with the most stake are the ones who are designated as validators. If a validator’s total stake falls below the top 100 then that validator loses its validator’s privileges. Over time, the maximum number of validators will increase, according to a predefined schedule:
+
+* **Year 0:** 100
+* **Year 1:** 113
+* **Year 2:** 127
+* **Year 3:** 144
+* **Year 4:** 163
+* **Year 5:** 184
+* **Year 6:** 208
+* **Year 7:** 235
+* **Year 8:** 265
+* **Year 9:** 300
+* **Year 10:** 300
+
+### Is there a minimum amount of ATOMs that must be staked to be a validator?
+
+There is no minimum. The top 100 validator candidates with the highest total stake (where total stake = self-bonded stake + delegators stake) are the one that become validators. 
+
+### How will delegators choose their validators?
+
+Delegators are free to choose validators according to their own subjective criteria. However, there are a number of objective criteria that can be expected to influence delegators decision when it comes to choosing their validators, namely:
+* **Commission rate:** Validators with low commissions will likely be more attractive.
+* **Amount of self-bonded ATOMs:** Number of ATOMs a validator self-bonded to its staking pool. A validator with higher amount of self-bonded ATOMs has more skin-in-the-game, making it more liable for its actions.
+* **Amount of delegated ATOMs:** Total number of ATOMs delegated to a validator. A high stake shows that the community trusts this validator, but it also means that this validator is a bigger target for hackers. Indeed, hackers are incentivized to hack bigger validators as they receive a reward proportionate to the stake of the validator they can prove to have compromised. We expect validators to become less and less attractive as their amount of delegated ATOMs grows.
```

Comment:
Let's remove the last sentence. `We expect validators ...`

Reply:
I will reformulate